controller/signUpController.js

```diff
@@ -0,0 +1,73 @@
+const User = require('../models/user')
```

Comment:
Is there a specific reason for using Camelcase sometimes, sometimes Uppercase and sometimes lowercase letters?

Reply:
naming convention